app/backend/usbforwardingtunnel.cpp

```
@@ -0,0 +1,225 @@
+    m_Remote = new QSslSocket(this);
+
+    /* Sunshine presents a self-signed certificate pinned at pairing time, so
+     * add it as a trust anchor on top of the client identity config. */
+    QSslConfiguration sslConfig = m_Config.sslConfiguration;
+    if (!m_Config.pinnedServerCertificate.isNull()) {
+        QList<QSslCertificate> caCertificates = sslConfig.caCertificates();
+        caCertificates.append(m_Config.pinnedServerCertificate);
+        sslConfig.setCaCertificates(caCertificates);
+        sslConfig.setPeerVerifyMode(QSslSocket::VerifyPeer);
+    }
+    m_Remote->setSslConfiguration(sslConfig);
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If Q_DISABLE_COPY is a macro then please configure it.

(unknownMacro)